cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -707,7 +724,7 @@ func Test_objectMover_backupTargetObject(t *testing.T) {
 				time.Sleep(time.Millisecond * 5)
```

Comment:
@sbueringer I mentioned this in Slack too, but I was able to fix the test on my machine by changing the duration here.
```suggestion
				time.Sleep(time.Millisecond * 50)
```

Reply:
Thx. Let's just try that and then we see over the next weeks/months if the flake still occurs.